src/platform/tls_openssl.c

```diff
@@ -1462,6 +1462,21 @@ CxPlatTlsSecConfigCreate(
         }
     }
 
+    if (CredConfig->CaCertificateFile) {
```

Comment:
On Schannel, since this isn't supported, we should return an error if this is specified (not_supported or invalid_parameter).